tests/features/camel-jcache/src/main/java/org/apache/karaf/camel/test/CamelJcacheEhRouteSupplier.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.karaf.camel.test;
+
+import org.apache.karaf.camel.itests.CamelRouteSupplier;
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+        name = "karaf-camel-jcache-eh-test",
+        immediate = true,
+        service = CamelRouteSupplier.class
+)
+public class CamelJcacheEhRouteSupplier extends AbstractCamelJcacheRouteSupplier {
+
+    public static final String PROVIDER_CLASS = "org.ehcache.jsr107.EhcacheCachingProvider";
```

Comment:
Can we use `EhcacheCachingProvider.class.getName()` instead?

Reply:
it's possible, but it requires to add a maven dependency to Ehcache and an import. What do you prefer ?

Reply:
I see, we can keep it like this but first we need to make sure that if we provide an invalid provider class, the test will fail